services/github/src/handlers/status.js

```diff
@@ -164,7 +165,28 @@ export async function statusHandler(message) {
       output_summary: outputSummary || taskDefinition.metadata.description,
       output_annotations: customCheckRunAnnotations,
     });
+
+    const artifactList = await this.queueClient.listArtifacts(taskId, runId, { limit: 50 });
+
     const output = githubCheck.output;
+
+    if(runs.length > 0) {
+      const taskExecutionTime = getTimeDifference(runs[runId]?.started, runs[runId]?.resolved);
+
+      output.addText(`Started: ${runs[runId]?.started !== undefined ? runs[runId]?.started : "n/a"}`);
```

Comment:
nitpick: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing 
```suggestion
      output.addText(`Started: ${runs[runId]?.started ?? "n/a"}`);
```